src/sentry/sentry_apps/api/endpoints/installation_external_issue_actions.py

```diff
@@ -56,34 +56,26 @@ def post(self, request: Request, installation) -> Response:
                 project_id__in=Project.objects.filter(organization_id=installation.organization_id),
             )
         except Group.DoesNotExist:
-            return Response(status=404)
+            raise SentryAppError(
+                message="Could not find the corresponding issue for the given groupId",
+                status_code=404,
+            )
 
         action = data.pop("action")
         uri = data.pop("uri")
 
-        try:
-            user = _extract_lazy_object(request.user)
-            if isinstance(user, User):
-                user = serialize_rpc_user(user)
+        user = _extract_lazy_object(request.user)
+        if isinstance(user, User):
+            user = serialize_rpc_user(user)
 
-            external_issue = IssueLinkCreator(
-                install=installation,
-                group=group,
-                action=action,
-                fields=data,
-                uri=uri,
-                user=user,
-            ).run()
-        except (APIError, ValidationError, APIUnauthorized) as e:
-            return Response({"error": str(e)}, status=400)
-        except Exception as e:
-            error_id = capture_exception(e)
-            return Response(
-                {
-                    "error": f"Something went wrong while trying to link issue. Sentry error ID: {error_id}"
-                },
-                status=500,
-            )
```

Comment:
is there a reason we're no longer capturing these errors?

Reply:
error handler in the base `IntegrationPlatformEndpoint` ([link](https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_apps/api/bases/sentryapps.py#L129-L149)) will do the catching for us